icons/notebook.svg

```diff
@@ -0,0 +1,18 @@
+<svg
```

Comment:
Not my favorite notebook, what about using `notebook-elastic` as the default notebook?

Reply:
See #1363.